index.html

```diff
@@ -199,15 +199,146 @@ <h1 class="header">What You Can Do at Technica</h1>
           </li>
         </ul>
       
-        <div class="tab-content">
+        <div class="tab-content unexpanded" onclick="expand(this)">
           <div id="friends" class="tab-pane fade in active show">
-            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            <div class="content-wrapper row">
+              <div class="col-lg-6">
+                <div class="content-heading">Mini Events</div>
+                <div class="content-text">Want to take a break from coding and have some fun? Mini Events are for you!
+                  Mini-Events are a fun way to take a brain break during the weekend, and allow you to meet new friends, get
+                  creative,
+                  and even make something you can take home! A few Mini Events we already have planned for this year include
+                  Open Mic
+                  Night,
+                  Family Feud, DIY Your Own Technica Swag, and Board Game Corner.</div>
+              </div>
+              <img class="content-img col-10 col-md-5 img-fluid" src="./img/minievents_cream.svg" />
+            </div>
+            <div class="row expand-btn">Read More
+              <img class="arrow-img" src="./img/arrow_down.svg" />
+            </div>
+
+            <div class="content-wrapper row expanded-content">
+              <div class="col-lg-6">
+                <div class="content-heading">Watercooler</div>
+                <div class="content-text">Here’s to a weekend of making new friends! A major part of the magic of Technica are the serendipitous 
+                  encounters hackers have in the hallways at 3am. Whether that’s bumping into someone or wandering over to the snack table 
+                  and striking up a conversation, hackers will often exit the weekend having made a friend. The virtual part of our hybrid 
+                  format this year means that it will be harder for these informal encounters to occur, and we’d like to help facilitate this 
+                  process via a virtual water cooler feature. Hackers will be able to join the virtual water cooler via the navigation bar in the 
+                  platform, and be instructed to join a shared Zoom call. From there, they can mingle with other attendees and be assigned to a 
+                  timed breakout room by the water cooler manager (a designated organizer) in order to chat informally with 3-5 other hackers and 
+                  a volunteer chaperone.
+                </div>
+              </div>
+              <img class="content-img col-10 col-md-5 img-fluid" src="./img/watercooler_cream.svg" />
```

Comment:
nit: can you remove "_cream" from all of the images names in the folder & in these `img` tags pls